<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EncryptionWindow</class>
 <widget class="QMainWindow" name="EncryptionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Simple-AES</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/icons/icon/lock.png</normaloff>:/icons/icon/lock.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>26</width>
    <height>26</height>
   </size>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextOnly</enum>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="main_frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1011</width>
      <height>611</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-color:rgb(255, 255, 255);
}

QPushButton {
	border: none;
}

</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <widget class="QFrame" name="content_frame">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>40</y>
       <width>641</width>
       <height>531</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	border-radius:10px;
	background-color:rgb(240, 240, 240);
	padding-left:10px;
	padding-right:10px;
}

QLineEdit:focus {
    border: 2px solid rgb(170, 170, 255); /* 获取焦点时的边框样式 */
}



QPushButton:hover {
	background-color:rgb(240, 240, 240);
}

#content_frame {
	border:3px solid rgb(170, 170, 255);
	border-radius:20px
}

</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="plain_text_input">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>40</y>
        <width>371</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="set_key_check">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Set Key</string>
      </property>
     </widget>
     <widget class="QLabel" name="encrpyt_text_label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>250</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Encrypted Text</string>
      </property>
     </widget>
     <widget class="QLabel" name="plain_text_label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Plain Text</string>
      </property>
     </widget>
     <widget class="QPushButton" name="generate_button">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>450</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color:white;
	background-color:rgb(85, 0, 127);
	border-radius:18px;
}

QPushButton::pressed {
	background-color:rgb(170, 170, 255);
}

</string>
      </property>
      <property name="text">
       <string>Generate</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="encrypted_text_input">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>250</y>
        <width>371</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QTextEdit {
	background-color:rgb(240, 240, 240);
	border:none;
	border-radius:10px;
	padding-left:10px;
	padding-right:10px;
}
QTextEdit:focus {
    border: 2px solid rgb(170, 170, 255); /* 获取焦点时的边框样式 */
}

QTextEdit QScrollBar::handle:vertical {
	background-color:rgb(170, 170, 255);
	border-radius:5px
}
</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="key_input">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>110</y>
        <width>371</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QTextEdit {
	background-color:rgb(240, 240, 240);
	border:none;
	border-radius:10px;
	padding-left:10px;
	padding-right:10px;
}

QTextEdit:focus {
    border: 2px solid rgb(170, 170, 255); /* 获取焦点时的边框样式 */
}


QTextEdit QScrollBar::handle:vertical {
	background-color:rgb(170, 170, 255);
	border-radius:5px
}</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="en_mode">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>380</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox{
	border:1px solid #242424;
	border-radius:3px;
	padding:2px;
	background:none;
	border-color: rgb(22,63,23);
	background-color: rgb(255, 255, 255);
	selection-background-color:#484848;
	selection-color:#DCDCDC;
}
 
 
QComboBox::down-arrow{
	image: url(:/icons/icon/down-arrow.png);
	width:20px;
	height:25px;
	right:0px;
}
 
QComboBox::drop-down{
	subcontrol-origin:padding;
	subcontrol-position:top right;
	width:20px;
	border-left-width:0px;
	border-left-style:solid;
}
 
QComboBox::drop-down:on{
	top:1px;
}
QComboBox QAbstractItemView::item{
	min-height:24px;
	min-width:20px;
	color: rgb(239, 239, 239);
}
QComboBox QAbstractItemView::item:selected
{	
    background-color: rgb(170, 170, 255);
}

</string>
      </property>
      <item>
       <property name="text">
        <string>normal encryption</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>double encryption</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>triple encryption</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="input_mode">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>380</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox{
	border:1px solid #242424;
	border-radius:3px;
	padding:2px;
	background:none;
	border-color: rgb(22,63,23);
	background-color: rgb(255, 255, 255);
	selection-background-color:#484848;
	selection-color:#DCDCDC;
}
 
 
QComboBox::down-arrow{
	image: url(:/icons/icon/down-arrow.png);
	width:20px;
	height:25px;
	right:0px;
}
 
QComboBox::drop-down{
	subcontrol-origin:padding;
	subcontrol-position:top right;
	width:20px;
	border-left-width:0px;
	border-left-style:solid;
}
 
QComboBox::drop-down:on{
	top:1px;
}
QComboBox QAbstractItemView::item{
	min-height:24px;
	min-width:20px;
	color: rgb(239, 239, 239);
}
QComboBox QAbstractItemView::item:selected
{	
    background-color: rgb(170, 170, 255);
}

</string>
      </property>
      <item>
       <property name="text">
        <string>string</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>binary</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="encrpyt_text_label_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>370</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>settings</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="vector_input">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>180</y>
        <width>371</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QTextEdit {
	background-color:rgb(240, 240, 240);
	border:none;
	border-radius:10px;
	padding-left:10px;
	padding-right:10px;
}
QTextEdit:focus {
    border: 2px solid rgb(170, 170, 255); /* 获取焦点时的边框样式 */
}


QTextEdit QScrollBar::handle:vertical {
	background-color:rgb(170, 170, 255);
	border-radius:5px
}</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="set_vector_check">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>180</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Initial Vector</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="menu_frame">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>251</width>
       <height>531</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border-radius:10px;
	background-color:rgb(255, 255, 255);
}

QFrame {
	background-color:rgb(210, 213, 255);
	border-radius:20px
}

</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="decryption_button">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton::hover {
	background-color:rgb(230, 230, 230)
}

QPushButton::pressed {
	background-color:rgb(224, 220, 240)
}</string>
      </property>
      <property name="text">
       <string>  Decryption</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/icons/icon/decryption.png</normaloff>:/icons/icon/decryption.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="encryption_button">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(95, 0, 147);
color:white;</string>
      </property>
      <property name="text">
       <string>  Encryption</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/icons/icon/encryption.png</normaloff>:/icons/icon/encryption.png</iconset>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="crack_button">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton::hover {
	background-color:rgb(230, 230, 230)
}

QPushButton::pressed {
	background-color:rgb(224, 220, 240)
}</string>
      </property>
      <property name="text">
       <string>Crack</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/icons/icon/spider.png</normaloff>:/icons/icon/spider.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="get_info_button">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>440</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color:white;
	background-color:rgb(119, 0, 255);
	border-radius:18px;
}

QPushButton::pressed {
	background-color:rgb(170, 170, 255);
}

</string>
      </property>
      <property name="text">
       <string>Encryption Info</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
